yasnippet.el

```diff
@@ -2736,9 +2738,8 @@ and `kill-buffer' instead."
                                  (point-max)
                                  (yas--template-expand-env yas--current-template))
              (when (and debug
-                        (require 'yasnippet-debug nil t)
-                        (fboundp 'yas-debug-snippet-vars))
-               (add-hook 'post-command-hook #'yas-debug-snippet-vars nil t))))
+                        (require 'yasnippet-debug nil t))
+               (yas-debug-snippet-create))))
```

Comment:
We'll need to add an argument here to `yas-debug-snippet-create`, or call `yas-debug-snippets` directly, because right now it passes `create` to `yas-debug-snippets` which puts an edebug breakpoint in `yas--snippet-parse-create` (which you only see if testing a snippet more than once, because we call the debug thing after the snippet is already expanded).

Reply:
Ah so that explains the breakpoint, which I though was mine. Will fix it.